### PR TITLE
Add doc comment to dns_error

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/checker.rs [hunk 1]
```diff
@@ -44,6 +44,7 @@ async fn do_request(client: &Client, url: &str) -> (RequestType, Result<Response
     (RequestType::Get, result)
 }
 
+/// Check if the reqwest error is a DNS error.
```
Comment (michellewzhang) [inline src/checker.rs:47]:
```suggestion
/// Check if the request error is a DNS error.
```

Comment (evanpurkhiser) [reply src/checker.rs]: The name of the library is actually [`reqwest`](https://docs.rs/reqwest/latest/reqwest/) lol

**Review by michellewzhang — APPROVED** [review]

File: src/checker.rs (hunk 1 above)
Comment (michellewzhang) [reply src/checker.rs]: oh whoops, sorry!!

Comment (michellewzhang) [reply src/checker.rs]: feel free to revert/make a new pr i can re-approve

Comment (evanpurkhiser) [reply src/checker.rs]: nah it's fine, doesn't really matter.

**Pull request merged**